frontend/src/views/Game.tsx

```diff
@@ -1,13 +1,44 @@
 import Editor from '@monaco-editor/react';
 import React from 'react';
+import { Problem, getProblems } from '../api/problem';
+import styles from './styles.module.css';
 
-function GamePage() {
-  return (
-    <div>
-      Game page
-      <Editor height="90vh" language="javascript" />
-    </div>
-  );
+interface GameState {
+  problems?: Problem[];
+}
+
+interface GameProps {}
+
+class GamePage extends React.Component<GameProps, GameState> {
+  constructor(props: GameProps) {
+    super(props);
+    this.state = {};
+  }
+
+  componentDidMount() {
+    getProblems().then((problems) => {
+      this.setState({ problems });
+    }).catch((error) => {
+      console.log(error);
```

Comment:
We may want to add some error validation to display an error component if something goes wrong, although that might come later since it'd apply to all the different game pages

Reply:
I'll add this to the trello board!

Reply:
This is more of an open discussion, both for @alankbi and @helenli8181, as I'm open to either. I wanted to propose resolving this immediately within this PR, because I think it may be good to say that when you introduce `catch` functions that are _going to stay_ (this isn't temporary code, like in PR #23), if the PR is not already bloated we add it to that same PR -- what are both of your thoughts?

Reply:
Bump @helenli8181 -- I think you forgot to respond to this before re-request?

Reply:
Working on it now :)